src/sentry/grouping/ingest/seer.py

```diff
@@ -28,7 +28,9 @@
 logger = logging.getLogger("sentry.events.grouping")
 
 
-def should_call_seer_for_grouping(event: Event, variants: dict[str, BaseVariant]) -> bool:
+def should_call_seer_for_grouping(
+    event: Event, variants: dict[str, BaseVariant], stacktrace_string: str
```

Comment:
```suggestion
    event: Event, variants: Mapping[str, BaseVariant], stacktrace_string: str
```

For arguments, it's recommended we prefer abstract types for function signatures:
https://typing.readthedocs.io/en/latest/reference/best_practices.html#arguments-and-return-types